client_test.go

```diff
@@ -49,6 +42,53 @@ func buildGoodResponse(data interface{}) ([]byte, error) {
 
 }
 
+func buildInternalServerErrorResponse(data interface{}) ([]byte, error) {
+	resp := &Response{
+		Status:		 "err",
```

Comment:
did you run go fmt on this. Seems like weird indentation.

Reply:
I did not... sorry, haven't been coding too much in go lately